Functions.Templates/Templates/SqlOutputBinding-CSharp/SqlOutputBindingCSharp.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Company.Function
+{
+    public static class SqlOutputBindingCSharp
+    {
+        // Visit https://github.com/Azure/azure-functions-sql-extension#Output-Binding-Tutorial to learn how to use this output binding
+        // and https://github.com/Azure/azure-functions-sql-extension#Output-Binding for more details
+        [FunctionName("SqlOutputBinding")]
+         public static Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
+            [Sql("table", ConnectionStringSetting = "SqlConnectionString")] out Object output,
+            ILogger log)
+        {
+            log.LogInformation("C# HTTP trigger with SQL Output Binding function processed a request.");
+
+            throw new NotImplementedException();
```

Comment:
Probably not useful to throw here...let's add some example code showing how one could use this with an actual object (I'd just define one in the template for simplicity)

Reply:
Added `TodoItem` example to the template